net.opentabletdriver.OpenTabletDriver.metainfo.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop-application">
+  <id>net.opentabletdriver.OpenTabletDriver</id>
+  <name>OpenTabletDriver</name>
+  <summary>A cross-platform open-source tablet driver</summary>
+  <developer>
+    <name>InfinityGhost</name>
+  </developer>
+  <metadata_license>GFDL-1.3</metadata_license>
+  <project_license>LGPL-3.0-or-later</project_license>
+  <update_contact>2835365572zty@gmail.com</update_contact>
+  <project_group>OpenTabletDriver</project_group>
+  <description>
+    <p>OpenTabletDriver is an open source, cross platform, user mode tablet driver. The goal of OpenTabletDriver is to be cross platform as possible with the highest compatibility in an easily configurable graphical user interface.</p>
```

Comment:
How is a driver going to work from inside a flatpak?

Reply:
It's a user space driver that doesn't need special permissions.